src/midi/midi_fluidsynth.cpp

```diff
@@ -869,28 +897,24 @@ std::string format_sf2_line(size_t width, const std_fs::path& sf2_path)
 
 MIDI_RC MidiHandlerFluidsynth::ListAll(Program* caller)
 {
-	// Find SoundFont from user config. FluidSynth may not be open so it
-	// must be done here.
-	auto* section = static_cast<Section_prop*>(control->GetSection("fluidsynth"));
-	const auto sf_spec = parse_soundfont_pref(section->Get_string("soundfont"));
-	const std_fs::path found_soundfont = find_sf_file(
-	        std::get<std::string>(sf_spec));
-
 	const size_t term_width = INT10_GetTextColumns();
 
 	auto write_line = [&](const std_fs::path& sf2_path) {
-		constexpr auto green = "[color=light-green]";
-		constexpr auto reset = "[reset]";
-
 		const auto line = format_sf2_line(term_width - 2, sf2_path);
-		const bool do_highlight = is_open &&
-		                          (selected_font == sf2_path.string());
+
+		const auto path_matches = current_sf2_path &&
+		                          (*current_sf2_path == sf2_path.string());
```

Comment:
Can you not do direct compares of two paths?  Why do you need the `.string()`?

Reply:
Might be a leftover from getting an earlier revision compile on MSVC. I vaguely remember I had to do `p1.string() == p2.string()` otherwise MSVC complained about it... But this works, so I'm really unsure. I'll remove `string()` and see what happens 🤷🏻

Reply:
Yeah, works 🤷🏻